proto/storage/runtime_filters.proto

```diff
@@ -39,3 +39,10 @@ message RuntimeFilteringConfiguration {
   repeated RuntimeFilter runtime_filters = 1;
   repeated ResourceCollection resource_collections = 2;
 }
+
+message RuntimeFilterData {
+  string id = 1; // @gotags: search:"Collection ID" sql:"pk"
+  RuntimeFilterFeatures feature = 2;
+  string status = 3;
+  string resource_collection_id = 4;
+}
```

Comment:
In the future it is likely that all fields will have their own columns and there won't be serialized columns.